www/js/app.js

```diff
@@ -383,7 +384,7 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('welcomeText').style.display = '';
         // Give the focus to the search field, and clean up the page contents
         $("#prefix").val("");
-        $('#prefix').focus();
+        document.getElementById('prefix').focus();
         $("#articleList").empty();
         $('#articleListHeaderMessage').empty();
```

Comment:
Perhaps you can do these two empty() statements as well. See https://youmightnotneedjquery.com/?support=ie11#empty .